contrib/endpoints/src/api_manager/context/request_context.cc

```diff
@@ -226,6 +226,18 @@ void RequestContext::FillCheckRequestInfo(
   info->allow_unregistered_calls = method()->allow_unregistered_calls();
 }
 
+void RequestContext::FillAllocateQuotaRequestInfo(
+    service_control::QuotaRequestInfo *info) {
+  FillOperationInfo(info);
+
+  info->client_ip = request_->GetClientIP();
+  info->method_name = this->method_call_.method_info->name();
+
+  // TODO(jaebong) need to set quota rule and metric rule
+  info->quota_rule_ = nullptr;
```

Comment:
This part was not clear when I created this PR. 
The changes are included in https://github.com/istio/proxy/pull/101. 
Can I change this after the PR merge?